operations/paymentrequests.md

```diff
@@ -0,0 +1,264 @@
+# Payment Requests
+
+## Get all payment requests
+
+Get all payment requests belonging to the specified [Customers](customers.md#customer). Note that this operation uses [Pagination](../guidelines/pagination.md).

```

Comment:
```suggestion
Get all payment requests belonging to the specified customer accounts. Note that this operation uses [Pagination](../guidelines/pagination.md).
```
I've suggested this change because in the description we talk about 'customers' but in the data we talk about 'accounts' and they don't tie up.